CMakeLists.txt

```diff
@@ -799,15 +805,21 @@ if(BUILD_TESTING)
     endif()
 
     if(BUILD_LIBSSL)
+      add_custom_target(
+          run_ssl_runner_tests
+          COMMAND cd ssl/test/runner &&
+                  ${GO_EXECUTABLE} test -timeout ${GO_TEST_TIMEOUT} -shim-path $<TARGET_FILE:bssl_shim>
+                    ${HANDSHAKER_ARGS} ${RUNNER_ARGS} ${AWS_LC_SSL_RUNNER_INDEX_FILTER}
+          WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+          DEPENDS bssl_shim handshaker fips_specific_tests_if_any
+          ${MAYBE_USES_TERMINAL})
+
       add_custom_target(
           run_tests
           COMMAND ${GO_EXECUTABLE} run util/all_tests.go -build-dir
                   ${CMAKE_BINARY_DIR}
-          COMMAND cd ssl/test/runner &&
-                  ${GO_EXECUTABLE} test -timeout ${GO_TEST_TIMEOUT} -shim-path $<TARGET_FILE:bssl_shim>
-                    ${HANDSHAKER_ARGS} ${RUNNER_ARGS}
           WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-          DEPENDS all_tests bssl_shim handshaker fips_specific_tests_if_any
+          DEPENDS all_tests run_ssl_runner_tests
```

Comment:
I assume you actually want to remove fips-specific tests here. ok.

Reply:
`fips_specific_tests_if_any` is empty when -DFIPS is not passed. No behavior change should happen here.

Reply:
Ahh lol I missed it here... https://github.com/awslabs/aws-lc/pull/351/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR814